chpl/chpl-service/src/main/java/gov/healthit/chpl/web/controller/CertificationIdController.java

```diff
@@ -148,7 +170,8 @@
     //
     // Verify whether one or more specific EHR Certification ID is valid or not.
     // **********************************************************************************************************
-    @ApiOperation(value = "Verify whether one or more specific EHR Certification IDs are valid or not.",
+    @Deprecated
+    @ApiOperation(value = "DEPRECATED.  Verify whether one or more specific EHR Certification IDs are valid or not.",
```

Comment:
I don't think we can deprecate this call. It's used by CMS to verify a huge number of IDs all at once, and I think they're sending more data than is allowed in the RequestHeaders of a GET, which is why we built it as a POST. Up for a conversation

Reply:
I thought that this was only used for a few IDs.  Should I remove the new version or just remove the deprecated annotation for the original method?